[INFO   ] Logger: Record log in /storage/emulated/0/code/util/DownWithYT/v1.1/scratchpad/.kivy/logs/kivy_22-10-01_1.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  9 2021, 08:02:40) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 build 1.9@4893595'>
[INFO   ] GL: OpenGL vendor <b'Imagination Technologies'>
[INFO   ] GL: OpenGL renderer <b'PowerVR Rogue GE8300'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <4096>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Text: Provider: sdl2
[INFO   ] Start: Started App
[WARNING] Base: Unknown <android> provider
[INFO   ] Base: Start application main loop
[INFO   ] GL: NPOT texture support is available
[INFO   ] USE: YT_Widget
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/code/util/DownWithYT/v1.1/scratchpad/scratchapp.py", line 90, in <module>
[WARNING] stderr:     scr.run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/app.py", line 950, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 582, in runTouchApp
[WARNING] stderr:     EventLoop.mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 347, in mainloop
[WARNING] stderr:     self.idle()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/base.py", line 387, in idle
[WARNING] stderr:     Clock.tick()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick
[WARNING] stderr:     self.post_idle(ts, self.idle())
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/clock.py", line 776, in post_idle
[WARNING] stderr:     self._process_events()
[WARNING] stderr:   File "kivy/_clock.pyx", line 616, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 649, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 645, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 218, in kivy._clock.ClockEvent.tick
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/uix/recycleview/__init__.py", line 257, in refresh_views
[WARNING] stderr:     lm.set_visible_views(indices, data, viewport)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/uix/recyclelayout.py", line 235, in set_visible_views
[WARNING] stderr:     new, remaining, old = self.recycleview.view_adapter.set_visible_views(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/uix/recycleview/views.py", line 406, in set_visible_views
[WARNING] stderr:     view = get_view(index, data[index],
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/uix/recycleview/views.py", line 262, in get_view
[WARNING] stderr:     view = self.create_view(index, data_item, viewclass)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/uix/recycleview/views.py", line 225, in create_view
[WARNING] stderr:     view = viewclass()
[WARNING] stderr:   File "/storage/emulated/0/code/util/DownWithYT/v1.1/scratchpad/scratchapp.py", line 37, in __init__
[WARNING] stderr:     self.yt = None
[WARNING] stderr:   File "kivy/properties.pyx", line 498, in kivy.properties.Property.__set__
[WARNING] stderr:   File "kivy/properties.pyx", line 542, in kivy.properties.Property.set
[WARNING] stderr:   File "kivy/properties.pyx", line 533, in kivy.properties.Property.set
[WARNING] stderr:   File "kivy/properties.pyx", line 1002, in kivy.properties.ObjectProperty.check
[WARNING] stderr:   File "kivy/properties.pyx", line 571, in kivy.properties.Property.check
[WARNING] stderr: ValueError: None is not allowed for YT_Widget.yt
